docs/competition/teams/process-docs/orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+---
+original:
+  authors: Sean Bedford
+  url: https://github.com/srobo/comp-team-coord-docs/ProcessDocs/orientationAndPitRules
+---
+## Welcome to Student Robotics 2017!
+
+## Pit Rules and Guidance
+The following rules must be observed at all times in the pits:
+
+1.	No battery charging. You must hand over your battery charger and spare batteries at reception. During the competition, visit Helpdesk (on the top floor, see map) to swap a discharged battery for a charged one.
+2.	No liquids except drinks to be present on the top floor. The carpet is expensive, and easily damaged by adhesives / paint / solvents.
+3.	No powered tools on the top floor (such as soldering irons). **Power tools should only be used in the downstairs area.**
+4.	Keep your robot in your pit when possible – we may need to find you quickly.
+5.	Please avoid blocking or restricting aisles through pits.
+6.	Please refrain from running any wireless access points, they will only interfere with the AP next to your pit
+
+## Network and Website Details
+A wireless network is available, connection details:
+
+* **SSID:** SR
+* **Password:** easyasabc
+
+You will find the following pages on the website useful:
+
+[http://srobo.org/comp/schedule](https://studentrobotics.org/comp/schedule) The schedule of matches.
+
+[http://srobo.org/comp/league](https://studentrobotics.org/comp/league) The current league scores.
+
+[http://srobo.org/comp/team](https://studentrobotics.org/comp/team) Statistics about your team and others.
+
+## Schedule
```

Comment:
I don't think it's useful listing a schedule here, as it changes quite a lot per year

Reply:
cb34c96 clarifies that this whole page is an example.